[32mINFO    [0m juju.model:model.py:2972 Waiting for model:
  mongodb-k8s/0 [idle] active: Primary
  mongodb-k8s/1 [idle] active: 
  mongodb-k8s/2 [idle] active: 
[32mINFO    [0m juju.model:model.py:2972 Waiting for model:
  mongodb-k8s/0 [idle] active: Primary
  mongodb-k8s/1 [executing] active: 
  mongodb-k8s/2 [executing] active:
[32mINFO    [0m pytest_operator.plugin:plugin.py:834 Model status:

Model  Controller          Cloud/Region        Version  SLA          Timestamp
test   microk8s-localhost  microk8s/localhost  3.4.4    unsupported  02:26:32Z

App            Version  Status       Scale  Charm          Channel      Rev  Address         Exposed  Message
application             active           1  application                   0  10.152.183.175  no       
mongodb-k8s             maintenance      3  mongodb-k8s                   0  10.152.183.121  no       backup started/running, backup id:'2024-08-15T02:26:28Z'
s3-integrator           active           1  s3-integrator  latest/edge   33  10.152.183.100  no       

Unit              Workload     Agent  Address       Ports  Message
application/0*    active       idle   10.1.109.146         
mongodb-k8s/0*    maintenance  idle   10.1.109.142         backup started/running, backup id:'2024-08-15T02:26:28Z'
mongodb-k8s/1     active       idle   10.1.109.143         
mongodb-k8s/2     active       idle   10.1.109.144         
s3-integrator/0*  active       idle   10.1.109.145         

[32mINFO    [0m pytest_operator.plugin:plugin.py:840 Juju error logs:

unit-new-mongodb-aws-1: 02:09:45 ERROR unit.new-mongodb-aws/1.juju-log new-mongodb-aws/1.is_unit_in_replica_set PyMongoError=No replica set members match selector "Primary()", Timeout: 1.0s, Topology Description: <TopologyDescription id: 66bd6368378ef2c3832177ef, topology_type: ReplicaSetNoPrimary, servers: [<ServerDescription ('new-mongodb-aws-1.new-mongodb-aws-endpoints', 27017) server_type: RSGhost, rtt: 0.0009732309999890276>]>
unit-new-mongodb-gcp-1: 02:24:24 ERROR unit.new-mongodb-gcp/1.juju-log new-mongodb-gcp/1.is_unit_in_replica_set PyMongoError=No replica set members match selector "Primary()", Timeout: 1.0s, Topology Description: <TopologyDescription id: 66bd66d7fc8d335f0794f394, topology_type: ReplicaSetNoPrimary, servers: [<ServerDescription ('new-mongodb-gcp-1.new-mongodb-gcp-endpoints', 27017) server_type: RSGhost, rtt: 0.0004364350002106221>]>

[32mINFO    [0m pytest_operator.plugin:plugin.py:904 Forgetting main...